cms/djangoapps/maintenance/views.py

```diff
@@ -0,0 +1,223 @@
+"""
+Views for the maintenance app.
+"""
+import logging
+from django.db import transaction
+from django.core.validators import ValidationError
+from django.core.urlresolvers import reverse_lazy
+from django.utils.decorators import method_decorator
+from django.utils.translation import ugettext_lazy as _
+from django.views.generic import View
+
+from edxmako.shortcuts import render_to_response
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.exceptions import ItemNotFoundError
+
+from contentstore.management.commands.utils import get_course_versions
+from util.views import require_global_staff
+
+
+log = logging.getLogger(__name__)
+
+# This dict maintains all the views that will be used Maintenance app.
+MAINTENANCE_VIEWS = {
+    'force_publish_course': {
+        'url': reverse_lazy('maintenance:force_publish_course'),
+        'name': _('Force Publish Course'),
+        'slug': 'force_publish_course',
+        'description': _('Sometimes the draft and published branches of a course can get out of sync.'
```

Comment:
You need spaces between each line in a multiline string if you want those spaces to appear. Also reformat:

```
'description': _(
    'Sometimes the draft and published branches of a course can get out of sync. '
    'This command resets the published branch of a course to point to the draft branch, '
    'effectively force publishing the course.'
),
```